README.md

```diff
@@ -1,3 +1,171 @@
 # npecl design
 
-more info as it is written...
+A work in progress design documentation for the "new pecl" iteration.
+
+Basis:
+
+ * [Discussion doc](https://docs.google.com/document/d/1_N0E9xo3jn9aKrIZHIbTYaY5lXw71BpSO6-it4cRpDo/edit)
+ * Internal PHPF dev meet kick-off to review (minutes?)
+
+## Assumptions
+
+ * npecl is the assumed name, although it is not confirmed
```

Comment:
Need to decide a name....

Reply:
My vote is for **:broccoli: Floret**

Reply:
Other community-suggested ideas:

 * :hot_pepper: Pepper ([src](https://twitter.com/simonhamp/status/1758446710295036412))
 * php-ext ([src](https://twitter.com/carlos_granados/status/1758445815209599182))
 * phpem/pem / PHP Extension Manager ([src](https://twitter.com/strandafili/status/1758447294234386765))
 * install-php-extension/php-install-extension/pie ([src](https://phpc.social/@ocramius@mastodon.social/111940752578558129) / [src](https://phpc.social/@psycodepath@mastodon.social/111940835012754835))